pandas/tests/resample/test_resampler_grouper.py

```diff
@@ -295,3 +296,35 @@ def test_median_duplicate_columns():
     result = df.resample("5s").median()
     expected.columns = result.columns
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.xfail(reason="marked as xfail for: #26564")
```

Comment:
giving a GH reference is good, but the rest of this doesnt give much information.  is there a single-line description that a reader would find informative?

can you add a "GH" in front of the "#" pls

ca